### PR TITLE
1765 - Update summary form font styles for better readability

### LINKED ISSUE #1765: Summary Form: Improve data readability by updating type hierarchy
**Is your feature request related to a problem? Please describe.**
With information dense screens, the `summary-form` can lose readability since there is only a small difference between label and content typography

- [ ] Design input by @kayiuho 
- [ ] Dev implementation

See: https://design.infor.com/code/ids-enterprise/latest/demo/components/expandablearea/example-index

**Describe the solution you'd like**
* Increasing font size for data points and/or using font weight to differentiate labels, data, and headings
* Needs to be updated in default theme but also supported in `uplift`

**Describe alternatives you've considered**
n/a

**Additional context**
image006

picture1

### PR DESCRIPTION
**Pull request by jmacaluso711**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
This PR will update the summary form/expandable area font styles for better legibility. I added a test page for edge cases where the markup might be slightly different.

**Related github/jira issue (required)**:
closes #1765 

**Steps necessary to review your pull request (required)**:
- pull branch
- go to http://localhost:4000/components/expandablearea/example-index

### CODE REVIEW

File: src/components/expandablearea/_expandablearea.scss [hunk 1]
```diff
@@ -115,9 +115,14 @@
 
   //Icon on the header
   .title {
-    @include font-size(16);
+    @include font-size(18);
```
Comment (nickwynja) [inline src/components/expandablearea/_expandablearea.scss]: This should probably use a token for consistent adjustment between soho and uplift themes.

**Comment by tmcconechy** [comment]
@jmacaluso711 when you remove draft and are all done i can reset the baseline for you for the tests.

**Comment by nickwynja** [comment]
I'm a bit concerned about vertical space taken up by the changes to `.expandable-area .title`.

May-16-2019 17-38-24

Maybe we just do the `.summary-form` weight changes for now and review the `.title` changes across a few different cases.
